sdk/bpf/scripts/install.sh

```diff
@@ -24,21 +24,37 @@ download() {
     ln -sf "$cache_directory/$dirname-$version.md" "$dirname-$version.md"
   }
 
-  if [[ -r $cache_dirname ]]; then
+  if [[ -r $cache_dirname../$dirname-$version.md ]]; then
```

Comment:
..?

Reply:
That's the marker used to determine if the download has completed, check it, not just that the dir exists, the dir may be partially filled

Reply:
oh, are you referring to the missing '/' or the fact that ".." is funky?

Reply:
IDK,all of it.  Just feels like a weird construct

Reply:
Cleaned it up a bit, did you have a different construct in mind?